### PR TITLE
Cybersecurity New Topic: Risk

### PR DESCRIPTION
**Pull request by jonchery**
### Description

Added new cybersecurity folder and concept, Risk

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/risk/risk.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:8]: If we could, can we add at least one more tag?

File: content/cybersecurity/concepts/risk/risk.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:19]: Remove extra line break

File: content/cybersecurity/concepts/risk/risk.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:25]: Remove the line breaks between the bullet points.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jonchery I did a first review and only had a couple of comments.

**Review by jonchery — COMMENTED** [review]
Completed request

**Review by SSwiniarski — APPROVED** [review]
@jonchery Looks good for a second review.

**Comment by SSwiniarski** [comment]
@jonchery, one more edit. Can we remove the trailing punctuation from the headers? The colon `:` will cause this to fail one of the checks.

**Review by jonchery — COMMENTED** [review]
Removed ":"

File: content/cybersecurity/concepts/risk/risk.md
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:23]:
```suggestion
A cybersecurity framework provides a common set of standards for organizations to assess and understand their overall security postures. Here are a few of the most common cybersecurity frameworks:

```

File: content/cybersecurity/concepts/risk/risk.md
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:24]: Do these have official sites?

File: content/cybersecurity/concepts/risk/risk.md
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:17]: Can we edit the heading to better reflect the info below?

**Review by yangc95 — COMMENTED** [review]
@jonchery I left a few comments

**Comment by SSwiniarski** [comment]
@jonchery, in the interests of getting this merged, I applied the above edits. Once @yangc95 reviews and approves, we can merge this.

**Comment by yangc95** [comment]
@jonchery live link: https://www.codecademy.com/resources/docs/cybersecurity/risk

Screenshot 2023-03-20 at 6 10 41 PM

cc: @SSwiniarski

**Pull request merged**
